### PR TITLE
Issue #10548: Migrate to Inline Config Parser in EqualsAvoidNullCheckTest

### PR DESCRIPTION
**Pull request by shashwatj07**
#10548

### CODE REVIEW

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/equalsavoidnull/InputEqualsAvoidNullTextBlocks.java [hunk 1]
```diff
@@ -10,17 +10,17 @@
 
 public class InputEqualsAvoidNullTextBlocks {
     public void equalsAvoid(String myString) {
-        if (myString.equals("stuff")) { // violation
+        if (myString.equals("stuff")) { // violation 'String literal expressions should be on the left side of an equals comparison.'
         }
-        if (myString.equals("""
-                stuff""")) { // violation
+        if (myString.equals(""" // violation 'String literal expressions should be on the left side of an equals comparison.'
```
Comment (shashwatj07) [inline src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/equalsavoidnull/InputEqualsAvoidNullTextBlocks.java:15]: Can't place comment here. What to do?

Comment (romani) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/equalsavoidnull/InputEqualsAvoidNullTextBlocks.java:15]: Place it on line above, put this case in issue where we will support comments previous lines or we will keep this case a bit isolated and keep old style of tests on it

Comment (shashwatj07) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/equalsavoidnull/InputEqualsAvoidNullTextBlocks.java]: done

**Review by timurt — APPROVED** [review]

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/equalsavoidnull/InputEqualsAvoidNullEnhancedInstanceof.java [hunk 2]
```diff
@@ -12,20 +12,21 @@
...
             }
         }
     }
 
     void foo (Object string) {
         if(string instanceof String string1) {
-            final boolean myBool = string1.equals("test1") // violation
-                    || string1.equals("test1.5") // violation
-                    || string1.equals("test2") || string1.equals("test3"); // violation 2x
+            final boolean myBool = string1.equals("test1") // violation 'String literal expressions should be on the left side of an equals comparison.'
```
Comment (romani) [inline src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/equalsavoidnull/InputEqualsAvoidNullEnhancedInstanceof.java:26]: long lines is attention killer of reviewer or future reader of this code.
It is not acceptable.
Please shorten a message, we do not need full/whole message, we need some short part of message to quickly understand violation.

**Review by romani — CHANGES_REQUESTED** [review]
items:

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/equalsavoidnull/InputEqualsAvoidNullEnhancedInstanceof.java (hunk 2 above)
Comment (shashwatj07) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/equalsavoidnull/InputEqualsAvoidNullEnhancedInstanceof.java]: done

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**